### PR TITLE
SAK-29382 - Fix session timeout warning message

### PR DESCRIPTION
**Pull request by SedueRey**

### CODE REVIEW

File: reference/library/src/morpheus-master/sass/modules/timeout-alert/_base.scss [hunk 1]
```diff
@@ -1,3 +1,10 @@
 #timeout_alert_body{
-
+	z-index: 1001;
+	background: $background-color;
+	position: absolute;
+	top: 10em;
+	left: 10em;
+	width: calc(100% - 20em);
```
Comment (buckett) [inline reference/library/src/morpheus-master/sass/modules/timeout-alert/_base.scss]: calc() doesn't have very good Android browser support, I'm not sure this is a big problem as more people move onto newer browsers. Also old copies of IE don't support it (but again not sure we care), http://caniuse.com/#feat=calc

Comment (SedueRey) [inline reference/library/src/morpheus-master/sass/modules/timeout-alert/_base.scss]: I can add a fallback in all calcs I have added in morpheus using a mixin instead a property.

From experience, do you think we'll have too much problems? by now, our users have had no problems with it but surely we should think in every user in Sakai.

**Comment by steveswinsburg** [comment]
Based on http://caniuse.com/#feat=calc I wouldn't bother adding a fallback. IE8 is running out of support and the Android version is pretty old too (<2% globally).

**Comment by SedueRey** [comment]
Ok, I'll update all calcs on morpheus-master

**Pull request merged**